src/Model/ModelManager.php

```diff
@@ -500,7 +500,10 @@ public function getSortParameters(FieldDescriptionInterface $fieldDescription, D
     {
         $values = $datagrid->getValues();
 
-        if ($fieldDescription->getName() === $values['_sort_by']->getName() || $values['_sort_by']->getName() === $fieldDescription->getOption('sortable')) {
+        if (isset($values['_sort_by'])
```

Comment:
This looks very complicated. You  can extra a meaningful method?

Reply:
I agree it's complicated, but I don't know the meaning of this check.
I just wanted to add a check for the definition of `_sort_by`.

So I can't really give meaning to this...

Reply:
@core23 Do you have a name for the check ? Because I don't.

I prefer to let this like it was and just adding an isset to not create a wrong name

Reply:
how about `isTheFieldAlreadySorted`? or similar, looks like it what it does.

Reply:
@core23 @franmomu Done.